tsconfig.json

```
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "declarationMap" : true,
+    "declarationMap": true,
     "esModuleInterop": true,
-    "lib": ["es6"],
+    "lib": ["es6", "dom"],
```

I think this is likely because [`AbortSignal` support was only added in Node 15](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal#browser_compatibility) and we’re targeting ES6. Looks like [`lru-cache` is polyfilling `AbortSignal` when necessary](https://github.com/isaacs/node-lru-cache/blob/a63ce28eacff77dfd30f4f6d62adcd361d8228ab/index.js#L8-L56), so might make more sense to avoid type checking dependencies rather than adding a lib to this package (this package targets the server, so adding `dom` seems risky).

```suggestion
    "lib": ["es6"],
    "skipLibCheck": true,
```

---

fixed, thanks for the info